CHANGELOG.md

```diff
@@ -562,7 +667,7 @@
 
     Y'all, we messed up and applied the wrong fix. The change which caused this problem was [this](https://github.com/cloudflare/wrangler/pull/1566).
 
-    [ObsidianMinor]: https://github.com/ObsidianMinor
+    [obsidianminor]: https://github.com/ObsidianMinor
```

Comment:
Did we change the tool that generates these or something? Seems weird it's changing the capitalization. Markdown is case insensitive though, so this should still work.